xbmc/addons/AddonDatabase.cpp

```diff
@@ -862,7 +868,30 @@ bool CAddonDatabase::Search(const std::string& search, VECADDONS& addons)
   return false;
 }
 
-bool CAddonDatabase::DisableAddon(const std::string &addonID, bool disable /* = true */)
+bool CAddonDatabase::DisableAddon(const std::string& addonID,
+                                  const AddonDisabledReason& disabledReason)
+{
+  try
+  {
+    if (!m_pDB)
+      return false;
+    if (!m_pDS)
+      return false;
+
+    std::string sql =
```

Comment:
`const std::string`... my favorite nitpick, as you know.

Reply:
😃 I had copy pasted that!